### PR TITLE
Standardizes z4, z5, z6 and z7 levels.

### PR DESCRIPTION
**Pull request by SamHPurp**
This is even more multi-station groundwork following up on a few previous PRs.

You won't see anything different on BoxStation/Cyberiad.

However, MetaStation now gets the updated z-levels that we enjoy for Cyberiad. The most notable of this is Z6, which on Meta is empty, and has some cool stuff on Box... and the Mining Asteroid, where the Mining Asteroid is seriously out of date.

This'll make things easier for future new stations also, IE: Ragnarok/Delta.... and will reduce workload for updating bits and bobs, and will limit the concern for all sorts of edge-cases.

The broken Telecomms remains a feature of MetaStation's z-levels....

This is near enough what TG do, whom house/run a wide map selection.

And I'm just trying to nail down a few bugs, but I was hoping to also include Z2 in this, but this might have to be in a future PR... this'd enable MetaStation to also enjoy the real nice Centcomm we now have.

:cl: Purpose2
tweak: MetaStation now has the same Mining Asteroid and various space levels as the Cyberiad.
/ :cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Wouldn't the only thing you need to do for zlevels that can be accessed by shuttle to be just make sure the shuttles/docking ports are the same? Some rotation details too maybe? I feel like that would be best just in case you manage to rotate certain zlevels instead of whole maps. That way you don't break anything between them

**Comment by SamHPurp** [comment]
@taukausanake that'd be the easy way, sure... make Meta's escape shuttle like Box...

I was hoping to do it more elegantly while keeping the integrity of the original design.

File: _maps/metastation.dm [hunk 1]
```diff
@@ -3,23 +3,23 @@ The /tg/ codebase currently requires you to have 7 z-levels of the same size dim
...
 z1 = station
 z2 = centcomm
 z3 = derelict telecomms satellite
-z4 = derelict station
+z4 = engineering ship
 z5 = mining
-z6 = empty space
-z7 = empty space
+z6 = russian derelict
+z7 = empty
```
Comment (FlattestGuitar) [inline _maps/metastation.dm]: Technically, Z7 is space ruins now.

Comment (SamHPurp) [inline _maps/metastation.dm]: ... sorted

**Comment by taukausanake** [comment]
@SamHPurp I meant the rotation thing that is done when the shuttle is hijacked by the syndicate. Can't you just do that for the horizontal shuttle so it flies vertically?

**Comment by SamHPurp** [comment]
@taukausanake Right, and thats the idea. There are bugs however in doing so, which I haven't been able to nail down yet.

If I override the safety-checks, it works 100% fine in what shows up, what rotates etc... but it keeps failing the safety checks of things being in the way, and I haven't worked out wtf is going on with that yet.

I think its an existing bug with the system, not something snowflakey unique.

If you have any suggestions/advice on that subject, I'd love to hear it.

**Comment by SamHPurp** [comment]
With Lavaland coming, there is even more need/importance to standardize ALL z-levels.

I'll renew my effort on fixing the shuttle-rotation code so that we can use Z2 as well....

**Comment by SamHPurp** [comment]
This has now been mostly subsumed into another PR.

**Pull request closed without merging**